sdk/storage/azure-storage-file-datalake/azure/storage/file/out/datalake/_data_lake_file_client.pyi

```diff
@@ -0,0 +1,17 @@
+# Stubs for datalake._data_lake_file_client (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from azure.storage.file.datalake._path_client import PathClient
+from typing import Any, Optional
+
+class DataLakeFileClient(PathClient):
+    def __init__(self, account_url: str, file_system_name: str, file_directory: str, file_name: str, credential: Optional[Any]=..., **kwargs: Any) -> Any: ...
+    @classmethod
+    def from_connection_string(cls: Any, conn_str: str, file_system_name: str, directory_name: str, file_name: str, credential: Optional[Any]=..., **kwargs: Any) -> DataLakeFileClient: ...
+    def create_file(self, content_settings: Optional[ContentSettings]=..., metadata: Optional[Dict[str, str]]=..., **kwargs: Any) -> Dict[str, Union[str, datetime]]: ...
+    def delete_file(self, **kwargs: Any) -> None: ...
+    def get_file_properties(self, **kwargs: Any) -> FileProperties: ...
```

Comment:
Hi @lmazuel ! It seems we want to return Dict for all get_*_properties based on your discussion with Anna @annatisch ? or we only want to return Dict for get account properties? 😃 I'm a little bit confused because the _blob_client is still returning BlobProperties for get_blob_properties. I will adjust the return type based on your confirmation!